src/app/routes/registration/request-claim/request-claim.component.scss

```diff
@@ -24,4 +24,14 @@
         }
       
   }
+
+  ::ng-deep .mat-checkbox-disabled:not(.mat-checkbox-checked) .mat-checkbox-frame {
```

Comment:
Just an information: When using ::ng-deep you create global styles, so you are loosing styles encapsulation. Which means that when this component is displayed it can override some styles from different components. So if you want encapsulation, it is good to add pseudo-class selector `:host`. Example: `:host ::ng-deep ...`